src/commands/appSettings/uploadAppSettings.ts

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { WebSiteManagementModels } from "azure-arm-website";
+import * as dotenv from 'dotenv';
+import { Uri } from "vscode";
+import { AppSettingsTreeItem, SiteClient } from "vscode-azureappservice";
+import { envFileName } from "../../constants";
+import { ext } from "../../extensionVariables";
+import * as workspaceUtil from '../../utils/workspace';
+import { getLocalEnvironmentVariables } from "./getLocalEnvironmentVariables";
+
+export async function uploadAppSettings(target?: Uri | AppSettingsTreeItem | undefined): Promise<void> {
+    const message: string = 'Select the local .env file to upload.';
```

Comment:
nit: Move message down closer to where it's actually used (inside the "else" case)